packages/ember-metal/tests/alias_test.js

```diff
@@ -108,5 +108,5 @@ function assertDirty(obj, callback, label) {
   let tag = tagFor(obj);
   let tagValue = tag.value();
   callback();
-  ok(!tag.validate(tagValue), label);
+  QUnit.config.current.assert.ok(!tag.validate(tagValue), label);
```

Comment:
Whats your take on this vs passing assert into assertDirty?

Reply:
just my own laziness I think

Reply:
I think there is just one occurrence of assertDirty so I say we hook it up and then merge this